Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md

```diff
@@ -0,0 +1,329 @@
+# Microsoft Sentinel Solutions Packaging Tool Guidance For CCP Connector
+
+To create a package using V3 packaging tool refer [Link to V3](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.md)
+
+## CCP connector Building Blocks:
+Every CCP connector will have 4 building blocks and should be specified in sequence given below:
+1. Data Connector Definition - Consider this an a UI page shown after deployment to users and contains instruction steps and connectorUiConfig section.
+2. Data Connectors (Poller) - This will contain details to connect to your REST API and has a poller details.
+3. Data Collection Rule - Specify, what data should be collected, how to transform that data, and where to send it.
+4. Tables (Optional) - Stores your data logs.
+
+## How to Use ?
+1. In Data input file under Azure-Sentinel/Solutions, open the solution you want to add a CCP connector.
+2. Open data input file and under "Data Connectors" array object add a path to new dataConnectorDefinition file.
+3. File name to dataConnectorDefinition should be meaningful and recommended to have a suffix "Definition" attached to the name at the end e.g. dataConnectorDefinition.json.
+4. Navigate to your "Solutions/*yourSolutionName*/Data Connectors" folder and create a new folder for ccp suffix at the end e.g: DataConnectorDefinitionName_ccp".
```

Comment:
This is not clear and users will not be able to understand properly. what is the data connector definition name_ccp? can we simply call the root folder for ccp as connectorname_ccp

Reply:
updated read me file and add full flow image in document